taxonomies/categories.txt

```diff
@@ -45799,37 +45934,46 @@ en:Fruits cereal bars
 fr:Barres de céréales aux fruits, Barre de céréales aux fruits
 it:Barrette di cereali con frutti
 nl:Ontbijtrepen met fruit
+ro:Batoane de cereale cu fructe
 
 <en:Fruits cereal bars
 en:Apricot cereal bars
 fr:Barres de céréales à l'abricot, Barre de céréales à l'abricot
 it:Barrette di cereali con albicocca
 nl:Graanrepen met abrikozen
 
+<en:Fruits cereal bars
+en:Strawberry cereal bars
+ro:Batoane de cereale cu căpșuni
```

Comment:
```suggestion
ro:Batoane de cereale cu căpșuni
fr:Barres de céréales aux fraises, Barre de céréales aux fraises
```

Reply:
Done. Added this translation in 9cd0284.